src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Math magicians
-        </a>
-      </header>
-    </div>
+    <Calculator />
   );
 }
 
```

Comment:
- [ ] Great job you have imported the Calculator.js in the App.js file, however, The App.js component should be class-based.
![storage1](https://user-images.githubusercontent.com/55635977/133318323-e3a5036c-7db4-43dc-a31e-c470edf9bfb6.png)

Reply:
@Emmyn5600 Thank you, I got it.